src/app/accounts/destiny-account.ts

```diff
@@ -42,7 +45,9 @@ const PLATFORM_LABELS = {
   [BungieMembershipType.BungieNext]: 'Bungie.net',
 };
 
-export const PLATFORM_ICONS = {
+export const PLATFORM_ICONS: Record<BungieMembershipType, string | IconDefinition> = {
```

Comment:
not into adding fake runtime data to satisfy the type.
if anything this should be Partial,
and then either
- the assertion that None, All, Demon, and Bungie will _never ever_ show up, should appear as a ! in the Account component
- an && check in Account component can make sure something was lookup'd

or just insert a question mark fa icon or somthing

Reply:
I made it complete so we'd be forced to add new platforms to this list by the typing. I can exclude the useless ones though.

Reply:
Actually trying to omit it is a huge pain.